lib/core/seleniumRunner.js

```diff
@@ -506,18 +507,22 @@ class SeleniumRunner {
    * @param {*} scriptsByCategory
    * @param {boolean} isAsync - is the script synchrously or async?
    */
-  async runScripts(scriptsByCategory, isAsync) {
+  async runScripts(scriptsByCategory, isAsync, type = ScriptType.COLLECT) {
     const categoryNames = Object.keys(scriptsByCategory);
     const results = {};
     for (let categoryName of categoryNames) {
       const category = scriptsByCategory[categoryName];
       try {
-        results[categoryName] = await this.runScriptInCategory(
+        const categoryResults = await this.runScriptsInCategory(
           category,
-          isAsync
+          isAsync,
+          type
         );
+        if (type === 'collect') {
```

Comment:
Sometimes we use `ScriptType.*`, sometimes raw strings.  Might as well be consistent with `ScriptType.*` if we're going to be heavy-weight.